Packages/install_flasher

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-2.0
+
+set -e
+for f in `ls ${VERSIONING}/*`
+  do
+    source $f
+  done
+if [ -f "customize.sh" ] ; then
+  source customize.sh
+fi
+cd Packages
+
+echo "** install_flasher **"
+set -x
+cd Flasher
+
+cp functions.sh init-eMMC-flasher-v3.sh umikaze-files partition.py /opt/scripts/tools/eMMC/
+sed -i 's/#enable_/enable_/' /boot/uEnv.txt
```

Comment:
This is dangerous, as I think there may be more than one line beginning with `enable`...

Reply:
I agree that this is dangerous. But that is what was present in the previous incarnation.

Personally, rather that editing some opaque file, I would rather see us replace the entire file with a copy that we have edited. That way, you can see the result in our repo and not have to guess what it is doing.

Reply:
The problem is that we then have to maintain it with whatever exact kernel version will be installed by the package manager. I think it's a better idea to make the sed keyword a longer one, to isolate a specific line we want.

Reply:
Yes, we have to maintain it. But, we also should be maintaining a specification of the kernel to be installed. If we change kernels, one of the things we MUST do is make sure that this patch is still appropriate. It's not really any more effort to update our cached copy of the file in the process.

Reply:
"install_flasher" removed (it will be back in a later PR) and changed rebased.

Any thing else about this portion of the changes?